2019/questions.md

```diff
@@ -386,9 +386,31 @@
 * Other
 >LH Note: who is supposed to get this question?
 
-41. Thank you for taking the time to answer our questions. If you would like a copy of the results, please give us your email address. We will only use this email address to communicate with you about this survey.
+41. To what degree do you perceive the following companies to be “good open source community citizens” in terms of their contributions, collaboration and leadership on open source projects and initiatives within the CNCF ecosystem? 
```

Comment:
I think the question can be broadened to reference all projects and initiatives related to cloud services.

Reply:
Change "companies" to  TODO Group members